sw/device/tests/sim_dv/rv_dm_ndm_reset_req.c

```diff
@@ -114,8 +114,8 @@ static test_register_t kReg[] = {
             .name = "PINMUX",
             .base = TOP_EARLGREY_PINMUX_AON_BASE_ADDR,
             .offset = PINMUX_WKUP_DETECTOR_CNT_TH_1_REG_OFFSET,
-            .write_val = 0x44,
-            .exp_read_val = 0x44,
+            .write_val = 0x01,
```

Comment:
The register won't accept any value different from 0x01 to be written to it. 
Is that some kind of filter?
Have this bahavior changed recently?

Reply:
This test program PINMUX.WKUP_DETECTOR_CNT_TH_1[https://docs.opentitan.org/hw/ip/pinmux/doc/index.html#Reg_wkup_detector_cnt_th_1] which has 8bits. So shouldn't be the case @msfschaffner can you confirm?

Reply:
I did some more tests and this is how the register is behaving:
![image](https://user-images.githubusercontent.com/65042207/187870351-dfdb1b60-8de4-4842-85c8-d405d0710be6.png)

Reply:
I am currently looking at the dump. 
Based on @msfschaffner 's comment(https://github.com/lowRISC/opentitan/issues/14198#issuecomment-1232005309)
There are some issue here. Let me follow up in this thread.

Reply:
@jdonjdon let us know if you need help to look at this.
I wonder if our multi-reg thing screwed up...

Reply:
Yeah let us know. Maybe there is a bug in the multireg variant of the async transition logic that we somehow missed...

Reply:
@engdoreis I make a [PR](https://github.com/lowRISC/opentitan/pull/14709) to fix this. The problem was this test use pin_regs.h from hw/ip not hw/top_earlgrey/ip.
With a new PR, this register can take 8bit value.

Reply:
Great, thanks for debugging that @jdonjdon!

Reply:
phew.. Thanks @jdonjdon!

Reply:
Thanks @jdonjdon, I removed the commit changing the register value from this PR, but I still think it is worth merging the refactor PR.